caps/eth.md

```diff
@@ -432,6 +481,13 @@ All three payload elements must contain an equal number of items.
 the byte size of `tx-type || tx-data` for typed transactions, and the size of the
 RLP-encoded `legacy-tx` for non-typed legacy transactions.
 
+The `cells` element is a bitmap marking which cell indices can be fetched from the sending 
+peer. For each bit set to one, the peer stores the cell at that index in all blobs of the 
+transaction. A bit must be set only if the peer has the cell at that index in all blobs of 
+the transaction. This field is only relevant for those entries that refer to blob 
+transactions. Blob transactions with the same `cells` field may be announced together in a 
```

Comment:
I see, the the intended notion is that the bitmap refers to all blobs of all transactions. Otherwise one would need separate NewPooledTransactionHashes messages for each transaction.